public/javascripts/global.js

```diff
@@ -129,14 +129,15 @@ function fetchQuiz(event) {
     var target = $(event.target);
     if (target.hasClass('quiz-btn')){
       var team = $('.selected-team'),
-          id = team.data('id');
+          id = team.data('team-id'),
```

Comment:
I forgot how this works. Is this the RB id or the API id? Don't we need both of them?